### PR TITLE
VIH-10125 upgrade to dotnet 8

### PR DESCRIPTION
**Pull request by shaed-parkar**
### Jira link (if applicable)

VIH-10125

### Change description ###

upgrade to dotnet 8

### CODE REVIEW

File: variables/shared.yaml [hunk 1]
```diff
@@ -84,3 +84,8 @@ variables:
     value: "vh-infra-core-${{ parameters.env }}-sql-username"
   - name: vhSqlPasswordSecret
     value: "vh-infra-core-${{ parameters.env }}-sql-password"
+
+  - name: dotnetVersion
+    value: '8.x'
+  - name: efDotnetToolVersion
```
Comment (oliver-scott) [inline variables/shared.yaml]: `efDotnetToolVersion` doesn't appear to be used, can it be removed? Don't think user API uses EF

**Review by oliver-scott — COMMENTED** [review]
one small suggestion otherwise looks good

File: variables/shared.yaml (hunk 1 above)
Comment (shaed-parkar) [reply variables/shared.yaml]: removed, good point

**Review by SwapnaVersion1 — APPROVED** [review]

**Review by oliver-scott — APPROVED** [review]

**Review by komaldabhi1509 — COMMENTED** [review]
Approved

**Pull request merged**